src/omnisharp/observers/OmnisharpLoggerObserver.ts

```diff
@@ -58,6 +59,30 @@ export class OmnisharpLoggerObserver {
                     this.logger.appendLine();
                 }
                 break;
+            case MessageType.OmnisharpServerEnqueueRequest:
```

Comment:
What do you think about putting all these cases in another method and invoking that method if the debugMode is true ?